### DISCUSSION TITLE
Webadmin plugin bare metal ?

### DISCUSSION THREAD

**Discussion by siliconhippy** [eduardogsilva/wireguard_webadmin#38] [Help and Support]
Just stumbled here ...looks cool 😎 

1. Is this for the webadmin package as plugin ?
2. Docker can be confusing with ports and paths. Is there a bare metal install for webadmin server ?
3. Will this work with Surfshark subscription? I plan to use this as a gateway for my LAN devices without any VPN client installs on devices !
Cheers,
Shippy.

**Comment by eduardogsilva** [comment]
Hello @siliconhippy !

When I launched the project, I wrote a guide on how to run Wireguard Webadmin manually. However, although it is possible, I don't think it's a good idea. You will have to put in some work to run it manually, and the update process will be quite tedious. 

If you want more information on how to get started, you can visit [this link](https://github.com/eduardogsilva/wireguard_webadmin/discussions/2), but I believe it's not worth the effort. If it's a requirement, I suggest looking for another solution designed to run directly without Docker.

The ports used by Wireguard Webadmin are declared in the `docker-compose.yml` file:

- 51820-51839/udp (For Wireguard instances)
- 8080-8089/tcp (Reserved for port forwarding if you use it)
- 80 and 443 for the Nginx container.

In any case, the application you described is not really the scenario that Wireguard Webadmin is intended to serve. To have a gateway that routes all your local network traffic to an external VPN, I suggest setting up a dedicated firewall for that, such as pfSense, or using a Mikrotik.

I'm available if you have any questions.
Cheers!
